website/content/api-docs/secret/identity/mfa/duo.mdx

```diff
@@ -5,18 +5,16 @@ description: >-
   The '/identity/mfa/method/duo' endpoint focuses on managing Duo MFA behaviors in Vault.
 ---
 
-## Configure Duo MFA Method
+## Create Duo MFA Method
```

Comment:
```suggestion
## Create Duo MFA method
```
Style guide issue: use sentence case for headings

Reply:
As I have a common response to all of the 24 similar suggestions in this review, I've put it as a separate top-level comment below, and marked the other 23 suggestions as resolved, to merge the conversation flow into one place.